ansible/roles/encoder/tasks/packages.yml

```diff
@@ -6,6 +6,7 @@
     - libnginx-mod-rtmp
     - icecast2
     - samba
+#   - i965-va-driver-shaders
```

Comment:
I didn't know wether to put this in or not because it may be differ between encoders

Reply:
Please add

Reply:
Done https://github.com/voc/cm/pull/15/commits/d951860af36a9d94d4f9c26a75c249297c952a33